src/main/service/CachingService.ts

```diff
@@ -51,13 +51,12 @@ export class CachingService {
    * @param placeId - place ID of the restaurant
    */
   async getLatLonValue(placeId: string): Promise<CachingObject> {
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
       this.client.get(placeId, (err: Error | null, data: string | null) => {
-        if (err) {
-          // todo: https://github.com/ubclaunchpad/foodies/issues/89
-          reject(err);
+        if (err || !data) {
+          return resolve(JSON.parse('{}'));
```

Comment:
This is equivalent right?

```suggestion
        if (err || !data) {
          return resolve({});
```

Reply:
I'm gonna keep the old version cause Typescript isn't accepting it!

`Argument of type '{}' is not assignable to parameter of type 'CachingObject | PromiseLike<CachingObject>'.   Property 'then' is missing in type '{}' but required in type 'PromiseLike<CachingObject>'`

Reply:
Hmm ok generally if any language is complaining about types, then we should try to properly fix it. I think the JSON.parse is a "hack" that should be avoided